### PR TITLE
Router definition in glossary

### LINKED ISSUE #32230: Term "retailers" seems out of context
### MDN URL

https://developer.mozilla.org/en-US/docs/Glossary/Routers

### What specific section or headline is this issue about?

Routers #1

### What information was incorrect, unhelpful, or incomplete?

It took me a while to guess what the term "retailers" refers to and I am still not really sure. My best guess it is companies that sell routing equipment. However I would never think of these as retailers. That's more like Best Buy or something.

### What did you expect to see?

Something that tells the reader what a retailer is.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/glossary/routers`
* MDN URL: https://developer.mozilla.org/en-US/docs/Glossary/Routers
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/glossary/routers/index.md
* Last commit: https://github.com/mdn/content/commit/ada5fa5ef15eadd44b549ecf906423b4a2092f34
* Document last modified: 2023-06-08T13:19:16.000Z

</details>

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #32230

Mentioning "retailer" is completely unnecessary.

### CODE REVIEW

File: files/en-us/glossary/routers/index.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ page-type: glossary-definition
...
-3. In the implementation of an {{Glossary('API')}} in a service layer, a router is a software component that parses a request and directs or routes the request to various handlers within a program. The router code usually accepts a response from the handler and facilitates its return to the requester.
+1. In computer networing, a router is a device that forwards {{Glossary("Packet", "data packets")}} between computer networks.
+5. For a {{Glossary('SPA', 'Single-page application')}}, a router is a library that decides what web page is presented by a given {{Glossary('URL')}}.
+   This middleware module is used for all URL functions, as these are given a path to a file that is rendered to open the next page.
+7. In the implementation of an {{Glossary("API", "Application Programming Interface (API))}}, a router is a software component that parses a request and directs or routes the request to various handlers within a program.
```
Comment (hamishwillee) [inline files/en-us/glossary/routers/index.md:14]:
```suggestion
2. For a {{Glossary('SPA', 'Single-page application')}}, a router is a library that decides what web page is presented by a given {{Glossary('URL')}}.
   This middleware module is used for all URL functions, as these are given a path to a file that is rendered to open the next page.
3. In the implementation of an {{Glossary("API", "Application Programming Interface (API))}}, a router is a software component that parses a request and directs or routes the request to various handlers within a program.
```

**Review by Josh-Cena — COMMENTED** [review]
The mentions of "network/application/service layer" is definitely necessary in the understanding, IMO. It's standard terminology and helps people with background to put this information into context.

File: files/en-us/glossary/routers/index.md [hunk 2]
```diff
@@ -8,9 +8,11 @@ page-type: glossary-definition
 
 There are three definitions for **routers** on the web:
 
-1. For the network layer, the router is a networking device that decides data {{Glossary('Packet')}}s directions. They are distributed by retailers allowing user interaction to the internet.
-2. For a {{Glossary('SPA', 'Single-page application')}} in the application layer, a router is a library that decides what web page is presented by a given {{Glossary('URL')}}. This middleware module is used for all URL functions, as these are given a path to a file that is rendered to open the next page.
-3. In the implementation of an {{Glossary('API')}} in a service layer, a router is a software component that parses a request and directs or routes the request to various handlers within a program. The router code usually accepts a response from the handler and facilitates its return to the requester.
+1. In computer networing, a router is a device that forwards {{Glossary("Packet", "data packets")}} between computer networks.
```
Comment (hamishwillee) [inline files/en-us/glossary/routers/index.md:11]:
```suggestion
1. In computer networking, a router is a device that forwards {{Glossary("Packet", "data packets")}} between computer networks.
```

**Comment by hamishwillee** [comment]
> The mentions of "network/application/service layer" is definitely necessary in the understanding, IMO. It's standard terminology and helps people with background to put this information into context.

@Josh-Cena Strongly disagree, but is not worth my time to discuss. I reverted to just removing the problematic text.

File: files/en-us/glossary/routers/index.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ page-type: glossary-definition
 
 There are three definitions for **routers** on the web:
 
-1. For the network layer, the router is a networking device that decides data {{Glossary('Packet')}}s directions. They are distributed by retailers allowing user interaction to the internet.
+1. For the network layer, the router is a networking device that decides data {{Glossary('Packet')}}s directions.
```
Comment (bsmth) [inline files/en-us/glossary/routers/index.md:11]: Minor one for you to consider:

```suggestion
1. For the network layer, the router is a networking device that decides where to direct data {{Glossary('Packet', 'Packets')}}.
```

**Review by bsmth — APPROVED** [review]
Thanks, Hamish. One suggestion for you, but leaving a +1 👍🏻

File: files/en-us/glossary/routers/index.md (hunk 3 above)
Comment (wbamberg) [reply files/en-us/glossary/routers/index.md:11]: I was going to make a similar suggestion, except why capitalize "packets"?

File: files/en-us/glossary/routers/index.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ page-type: glossary-definition
 
 There are three definitions for **routers** on the web:
 
-1. For the network layer, the router is a networking device that decides data {{Glossary('Packet')}}s directions. They are distributed by retailers allowing user interaction to the internet.
+For the network layer, the router is a networking device that decides where to direct {{Glossary('Packet', 'data packets')}}.
```
Comment (hamishwillee) [inline files/en-us/glossary/routers/index.md:11]: Arggg
```suggestion
1. For the network layer, the router is a networking device that decides where to direct {{Glossary('Packet', 'data packets')}}.
```

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**